protostar/commands/cairo1_commands/test_rust_command.py

```diff
@@ -33,8 +40,18 @@ def arguments(self):
         ]
 
     async def run(self, args: Namespace):
+        contract_paths: dict[str, list[str]] = {}
+        for (
+            contract_name
+        ) in self._contract_path_resolver.configuration_file.get_contract_names():
+            contract_paths[contract_name] = [
+                str(path)
+                for path in self._contract_path_resolver.configuration_file.get_contract_source_paths(
```

Comment:
I'm not sure about using `configuration_file` from ContractPathResolver like that. I'd be better to directly inject a correct `ConfigurationFile` as a `TestRustCommand` dependency and use that instead.